src/Command/DispatchCommand.php

```diff
@@ -13,7 +13,6 @@
 
 namespace App\Command;
 
-use Doctrine\Common\Inflector\Inflector;
```

Comment:
Is this used somewhere else, can this dependency be dropped on composer.json?

Reply:
Done